<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>intersection_Dialog</class>
 <widget class="QDialog" name="intersection_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>交集</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;黑体&quot;;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="File1_lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="File1_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>文件1：</string>
   </property>
  </widget>
  <widget class="QToolButton" name="File1_toolButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="File2_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>文件2：</string>
   </property>
  </widget>
  <widget class="QToolButton" name="File2_toolButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="File2_lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="File1_col_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>列1：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="File1_spinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="File2_col_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>列2：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="File2_spinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>461</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>输出</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>471</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>文件1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>文件2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>所有文件</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
